### PR TITLE
Return early for IPv6 DNS queries in east-west domain

### PR DESCRIPTION
**Pull request by linki**
Might solve issues using alpine.

### CODE REVIEW

File: delivery.yaml [hunk 1]
```diff
@@ -52,6 +52,8 @@ pipeline:
           env:
           - name: E2E_NODE_OS
             value: ubuntu
+          - name: E2E_SKIP_CLUSTER_UPDATE
```
Comment (aermakov-zalando) [inline delivery.yaml]: Why?

Comment (linki) [reply delivery.yaml]: Just to ensure e2e is running against the new config.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by linki** [comment]
:+1:

**Pull request merged**